configs/tsconfig/base.json

```diff
@@ -15,7 +15,8 @@
     "preserveWatchOutput": true,
     "skipLibCheck": true,
     "strict": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "allowJs": true
```

Comment:
```cmd
0:0  error  Parsing error: "parserOptions.project" has been provided for @typescript-eslint/parser. The file was not found in any of the provided project(s): eslint.config.mjs
```

By default, tsconfig does not include .js files. This behavior caused the error:
"The file was not found in any of the provided project(s): eslint.config.mjs."
We resolved this issue by adding allowJs: true, which made it possible to treat .js files as part of the project.

ref: https://typescript-eslint.io/troubleshooting/typed-linting#i-get-errors-telling-me--was-not-found-by-the-project-service-consider-either-including-it-in-the-tsconfigjson-or-including-it-in-allowdefaultproject

Reply:
There are `.js` file extensions in [tseslint.configs.strictTypeChecked](https://github.com/toss/suspensive/pull/1326/files#diff-b30f7315dea69043e473e61637bf36479d298da8a0f1cdac60915633a448fba7R29). Could this be the reason for the parsing error?

Reply:
I'm wondering if the purpose of ESLint is to lint only TypeScript files. If that's the case, couldn't we remove the JavaScript-related extensions from the [scripts](https://github.com/toss/suspensive/blob/main/configs/eslint-config/package.json#L27)?

```diff
-"ci:eslint": "eslint \"**/*.{js,jsx,cjs,mjs,ts,tsx,cts,mts}\"",
+"ci:eslint": "eslint \"**/*.{ts,tsx,cts,mts}\"",
```

Reply:
I respect your opinion, and I believe that excluding checks for .js files in ESLint settings might be a somewhat regrettable choice. ESLint provides various rules that help maintain code conventions, and these rules still hold significant meaning for files that do not use TypeScript. For example, rules like import/order and prettier can greatly enhance code readability and maintain consistency within the project. From this perspective, I think it is more desirable to apply ESLint checks to all files.

Reply:
> There are .js file extensions in [tseslint.configs.strictTypeChecked](https://github.com/toss/suspensive/pull/1326/files#diff-b30f7315dea69043e473e61637bf36479d298da8a0f1cdac60915633a448fba7R29). Could this be the reason for the parsing error?

If we remove the .js extension from tseslint.configs.strictTypeChecked, the error will disappear, but this means that ESLint will not be applied to JavaScript files at all, which I think is not an ideal solution.

Reply:
> I believe that excluding checks for .js files in ESLint settings might be a somewhat regrettable choice. ESLint provides various rules that help maintain code conventions, and these rules still hold significant meaning for files that do not use TypeScript.

I agree absolutely!

Reply:
I want to follow this problem as @kangju2000 way. I agree both opinion by you guys, but I see this error! so I made this commit [8408867](https://github.com/toss/suspensive/pull/1326/commits/84088673b707a0a769e06f38042e1ef8bfdc0f1a)
Could you make new Pull Request if you want to check js,jsx,mjs,cjs at that time?

<img width="1024" alt="image" src="https://github.com/user-attachments/assets/c3b8e0b1-0bbe-41a2-9fe5-58fa1756b086">